README.adoc

```diff
@@ -114,32 +115,39 @@ mapping and registers a `MyConfiguration` bean.
 
 `Application` includes a `main()` method that autowires an instance of `MyConfiguration`.
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_mainhead.adoc[]
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_with_both.adoc[]
+== Run the Application
 
-As our `Application` implements `CommandLineRunner`, the `run` method is invoked automatically when boot
-starts. You should see something like this:
-....
+You can run the main method through your IDE.
+Alternatively, the `./gradlew bootRun` and `./mvnw spring-boot:run` commands will launch the application.
+
+As our `Application` implements `CommandLineRunner`, the `run` method is invoked automatically when boot starts. You should see the output:
+
+----
 ----------------------------------------
 Configuration properties
         example.username is demouser
         example.password is demopassword
 ----------------------------------------
-....
+----
 
-Now start your application with the `cloud` profile activated. You should see something like this:
-....
+Now start the application using the `cloud` profile.
+This can be accomplished in Gradle with the `./gradlew bootRun --args='--spring.profiles.active=cloud'` command or in Maven with the `./mvnw spring-boot:run -Dspring-boot.run.arguments="--spring.profiles.active=cloud"` command.
```

Comment:
Change `This can be accomplished` to `You can do so` (not passive and consistent with how we give instructions).